plugins/parsers/registry_test.go

```diff
@@ -16,11 +16,12 @@ import (
 
 func TestRegistry_BackwardCompatibility(t *testing.T) {
 	cfg := &parsers.Config{
-		MetricName:        "parser_compatibility_test",
-		CSVHeaderRowCount: 42,
-		XPathProtobufFile: "xpath/testcases/protos/addressbook.proto",
-		XPathProtobufType: "addressbook.AddressBook",
-		JSONStrict:        true,
+		MetricName:         "parser_compatibility_test",
+		CSVHeaderRowCount:  42,
+		XPathProtobufFile:  "xpath/testcases/protos/addressbook.proto",
+		XPathProtobufType:  "addressbook.AddressBook",
+		JSONStrict:         true,
+		AvroSchemaRegistry: "https://localhost:8081",
```

Comment:
Please skip backward compatibility test for `avro` as it doesn't need to be backward compatible.